.github/workflows/sca-scan.yml

```
@@ -6,10 +6,18 @@ jobs:
   security-sca:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@master
+      - name: Checkout repository
+        uses: actions/checkout@master
+      - name: Setup .NET Core @ Latest
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: "7.0.x"
```

Suggest use `8.0.x` since `7.0` has out of support

```suggestion
          dotnet-version: "8.0.x"
```

---

Thanks for the suggestion. @WeihanLi We'll take this seperately.